### PR TITLE
i18n(zh-cn): update `view-transitions.mdx`

### PR DESCRIPTION
**Pull request by liruifengv**
#### Description (required)

update `view-transitions.mdx`

#### Related issues & labels (optional)

- https://github.com/withastro/docs/pull/9224

### CODE REVIEW

File: src/content/docs/zh-cn/guides/view-transitions.mdx [hunk 1]
```diff
@@ -626,14 +626,41 @@ Astro 的视图过渡动画 API 生命周期事件顺序如下：
...
 </script>
 ```
 
+#### 构建自定义交换函数
+
+<p><Since v="4.15.0" /></p>
+
+`astro:transitions/client` 模块的 `swapFunctions` 对象提供了五个处理特定交换相关任务的工具函数,包括处理文档属性、页面元素和脚本执行。这些函数可以直接用于定义自定义的交换实现。
+
+以下示例演示了如何使用这些函数重新创建 Astro 的内置交换实现:
```
Comment (Nin3lee) [inline src/content/docs/zh-cn/guides/view-transitions.mdx:643]:
```suggestion
`astro:transitions/client` 模块的 `swapFunctions` 对象提供了五个处理特定交换相关任务的工具函数，包括处理文档属性、页面元素和脚本执行。这些函数可以直接用于定义自定义的交换实现。

以下示例演示了如何使用这些函数重新创建 Astro 的内置交换实现：
```

**Review by Nin3lee — CHANGES_REQUESTED** [review]
有些乱码和英文标点符号需要勘误~

**Review by Nin3lee — APPROVED** [review]
LGTM~

File: src/content/docs/zh-cn/guides/view-transitions.mdx [hunk 2]
```diff
@@ -626,14 +626,41 @@ Astro 的视图过渡动画 API 生命周期事件顺序如下：
...
+  swapFunctions.deselectScripts(doc);
+  swapFunctions.swapRootAttributes(doc);
+  swapFunctions.swapHeadElements(doc);
+  const restoreFocusFunction = swapFunctions.saveFocus();
+  swapFunctions.swapBodyElement(doc.body, document.body);
+  restoreFocusFunction();
+};
+...
+  event.swap = () => mySwap(event.newDocument);
+...
```
Comment (yanthomasdev) [inline src/content/docs/zh-cn/guides/view-transitions.mdx:659]:
```suggestion
import { swapFunctions } from 'astro:transitions/client';

// 用 doc 替换 window.document
function mySwap(doc: Document) {
  swapFunctions.deselectScripts(doc);
  swapFunctions.swapRootAttributes(doc);
  swapFunctions.swapHeadElements(doc);
  const restoreFocusFunction = swapFunctions.saveFocus();
  swapFunctions.swapBodyElement(doc.body, document.body);
  restoreFocusFunction();
};

...
  event.swap = () => mySwap(event.newDocument);
...
```

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good job! 🙌

**Pull request merged**